common/datavalues/src/data_type.rs

```diff
@@ -259,6 +259,22 @@ pub fn numerical_arithmetic_coercion(
     }
 }
 
+#[inline]
+pub fn numerical_singed_coercion(val_type: &DataType) -> Result<DataType> {
```

Comment:
single ?

Reply:
fixed, PTAL @sundy-li

Reply:
`numerical_signed_coercion` convert unsigned data type to signed